.github/abstract/Type Erasure.md

```diff
@@ -0,0 +1,4 @@
+Sometimes it may be desireable to explicitly remove type information. Often times this is when you want to pass around types with complicated generic signatures. SwiftCurrent, much like many of Apple's standard libraries, ships with several type erasers for your convenience. 
```

Comment:
Grammarly

```suggestion
Sometimes it may be desirable to explicitly remove type information. Often this is when you want to pass around types with complicated generic signatures. SwiftCurrent, much like many of Apple's standard libraries, ships with several type erasers for your convenience. 
```